boards/arm/nrf54l15pdk_nrf54l15/Kconfig.defconfig

```diff
@@ -10,7 +10,7 @@ config BT_CTLR
 	default BT
 
 config ROM_START_OFFSET
-	hex
```

Comment:
Why is the type hint removed?

Reply:
suggested in the Upstream PR: https://github.com/zephyrproject-rtos/zephyr/pull/68393#discussion_r1520950055